README.md

```diff
@@ -80,29 +80,71 @@ Think of WebUI like a WebView controller, but instead of embedding the WebView c
 
 ## Documentation
 
- - [Online Documentation - C](https://webui.me/docs/#/c_api)
- - [Online Documentation - C++](https://webui.me/docs/#/cpp_api)
+- [Online Documentation - C](https://webui.me/docs/#/c_api)
+- [Online Documentation - C++](https://webui.me/docs/#/cpp_api)
```

Comment:
Removing the preceding white space makes the formatting uniform with the `## Features` section in the readme.

My formatter was complaining about this, I hope it's okay to include it here.